File: 1erdepartamental/python/15.recursividad_fractales/main.py
"""
Programa: Recursividad Fractales
Demuestra generación recursiva de fractales
"""

#Imporate de las clases 
from core.fractales import FractalesRecursivos
#Punto de entrada del programa 
if __name__ == "__main__":
    fractales = FractalesRecursivos()

    fractales.sierpinski(1, 8)
    print()

    print("Conjunto de Cantor (nivel 1):")
    fractales.cantor(1, 27)

File: 1erdepartamental/c++/15.recursividad_fractales/main.cpp
#include <iostream>
#include "core/FractalesRecursivos.h"

int main() {
    FractalesRecursivos fractales;
    
    fractales.sierpinski(1, 8);
    std::cout << std::endl;
    
    std::cout << "Conjunto de Cantor (nivel 1):" << std::endl;
    fractales.cantor(1, 27);
    
    return 0;
}
